app/models/seance.rb

```diff
@@ -5,4 +5,12 @@ class Seance < ApplicationRecord
 
   validates :movie_id, presence: true
   validates :hall_id, presence: true
+
+  def confirmation_deadline
+    datetime - 30.minutes
+  end
+
+  def after_confirmation_deadline?
+    Time.now.after?(confirmation_deadline)
```

Comment:
I would use `Time.current` instead of `Time.now`

Reply:
You're right! ❤️  I just found this:
https://discuss.rubyonrails.org/t/time-now-vs-time-current-vs-datetime-now/75183

Reply:
Please check your time_zone before you do any changes. We are living in `GMT+0200` and my application was set to `config.time_zone = 'UTC'` which was `GMT+0000` and sideqik was using the correct time zone. I had to set the time zone up like the following:
```
class Application < Rails::Application
  config.time_zone = 'Warsaw'
end
```